FBTweak/_FBTweakCollectionViewController.m

```diff
@@ -10,12 +10,12 @@
 #import "FBTweakCollection.h"
 #import "FBTweakCategory.h"
 #import "FBTweak.h"
+#import "_FBKeyboardManager.h"
+#import "_FBTweakArrayViewController.h"
 #import "_FBTweakCollectionViewController.h"
-#import "_FBTweakTableViewCell.h"
 #import "_FBTweakColorViewController.h"
 #import "_FBTweakDictionaryViewController.h"
-#import "_FBTweakArrayViewController.h"
-#import "_FBKeyboardManager.h"
+#import "_FBTweakTableViewCell.h"
```

Comment:
Didn't we say that we wound't do any cosmetic changes to their code?

Reply:
right, reverted this